ballerina/tests/test.bal

```diff
@@ -0,0 +1,316 @@
+    // Assertions
+   test:assertNotEquals(response.records,null, msg = "Expected 'records' array to be non-empty");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetClassificationLevels() returns error? {
+    // Invoke the function without query parameters.
+    DataClassificationLevelsBean response = check jira->/rest/api/'3/classification\-levels();
+
+    // Check the response status code for both live and mock tests
+    test:assertTrue(response?.classifications is json[], msg = "Expected 'classifications' key to be present in the response.");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetDashboards() returns error? {
+    // Invoke the function without query parameters.
+    PageOfDashboards response = check jira->/rest/api/'3/dashboard();
+
+    // Check that the `dashboards` key exists and is an array in the response
+    test:assertTrue(response?.dashboards is Dashboard[], msg = "Expected 'dashboards' key to be present in the response and to be an array.");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testCreateDashboard() returns error? {
+
+    // Call the post function
+    Dashboard response = check jira->/rest/api/'3/dashboard.post(payload={
+        name: "Ansak's dashboard",
+        description: "A dashboard to help auditors identify sample of issues to check.",
+        sharePermissions: [
+        ],
+        editPermissions: []
+    });
+        test:assertNotEquals(response.id, null, msg = "Dashboard id should not be empty.");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetDefaultShareScope() returns error? {
+    // Call the get default share scope function
+    DefaultShareScope response = check jira->/rest/api/'3/filter/defaultShareScope();
+
+    // Assertion: Check that 'scopeType' exists in the response
+        test:assertNotEquals(response.scope,"", msg = "Expected 'scope' to have a value");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testPutDefaultShareScope() returns error? {
+    // Call the put function
+    DefaultShareScope response = check jira->/rest/api/'3/filter/defaultShareScope.put(payload = {scope:"AUTHENTICATED" });
+
+    // Assertion: Check that the response contains the expected scope value
+    
+        test:assertEquals(response.scope, "AUTHENTICATED", msg = "Expected 'scope' to be 'AUTHENTICATED'");
+}
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetFavouriteFilters() returns error? {
+
+    // Call the get function without any query parameters
+Filter[]|error response = jira->/rest/api/'3/filter/favourite();
+
+if (response is Filter[]) {
+    // Verify live response has at least one Filter object
+    test:assertNotEquals(response.length(), 0, msg = "No Favourite Filters.");
+} else {
+    test:assertFail(msg = "Error occurred while fetching favourite filters: " + response.toString());
+}
+}
```

Comment:
Please fix the formatting issues.

Reply:
And the same comment is applicable here as well.
https://github.com/ballerina-platform/module-ballerinax-jira/pull/5/files#r1836496330